package.json

```diff
@@ -71,6 +71,16 @@
     "typescript": "^3.3.3",
     "ws": "^6.2.0"
   },
+  "files": [
+    "LICENSE",
```

Comment:
I would remove `LICENSE`, `changelog.md`, and `readme.md` since they are always included by default.  I don't see any benefit to duplicating these in the `files` section.